fix: code style according PEP8 and endpoints naming

hchocobar opened the pull request:
@alesanchezr 
Buen día!
1. Propongo cambios en los archivos .py para adecuarlos a las PEP8 (identación de listas, dos lineas en blancos entre funciones)
2. Propongo realizar los endpoints en plural, para adecuarlos a la norma de nombres para endpoints.
3. Cambié el preview.png para adecuarlo a estos cambios.

hchocobar committed:
fix: code style accordin PEP8

marcogonzalo commented on src/test_solution.py:
```diff
@@ -1,111 +1,116 @@
-	})
+    response = client.post('/members', json={"first_name": "Tommy",
+                                             "id": 3443,
+		                                     "age": 23,
```
@hchocobar aquí veo que intentaste generar un poco de indentación pero no ha salido bien. Mi sugerencia:

```
    response = client.post('/members', json={
        "first_name": "Tommy",
        "id": 3443,
	"age": 23,
        "lucky_numbers": [34,65,23,4,6]
    })
```

marcogonzalo replied:
O como la línea 37, que ha quedado mejor.

hchocobar replied:
@marcogonzalo, el motivo es identación. 
Esta es una manera de escribir código en Python según PEP8 (no es la única)
Esta es la documentación: https://peps.python.org/pep-0008/#code-lay-out
![imagen](https://github.com/breatheco-de/exercise-family-static-api/assets/45291267/4cf1d63c-4dd7-4451-b28e-bbf4dfcfcf90)

marcogonzalo replied:
Sí, pero no está bien alineada de todos modos. Fíjate que hay unas lineas mucho más adentro y no a la misma altura de los anteriores. En el bloque de la Línea 37 sí están alineados, todos, en este bloque no.

hchocobar replied:
Esa es la idea Marco, alinear a la apertura del curly bracket de `json={`

Pero no identar a la altrua de `client.post(`

hchocobar committed:
Update test_solution.py

Corrección en lineas 30 y 31
Se reemplazan tab x espacios, esto soluciona la identación.

hchocobar committed:
Merge pull request #1 from hchocobar/hchocobar-patch-1

Update test_solution.py
